pandevice/__init__.py

```diff
@@ -66,10 +66,17 @@ def getlogger(name=__name__):
 # Enumerator type
 def enum(*sequential, **named):
     enums = dict(zip(sequential, range(len(sequential))), **named)
-    reverse = dict((value, key) for key, value in enums.iteritems())
+    reverse = dict(((v, k) for (k, v) in enums.items()))
     enums['reverse_mapping'] = reverse
     return type('Enum', (), enums)
 
+def isstring(arg):
+    try:
+        return isinstance(arg, basestring)
+    except NameError:
+        return isinstance(arg, str) or isinstance(arg, bytes)
```

Comment:
There are a few reasons why I am hesitant to do this, even though the tests still pass. I think its imperative that isstring() returns the same value given the same argument regardless of whether it runs on Python2 or Python3.
To do that I could either make isstring check if arg is a Python2 basestring (current implementation) or a Python3 string. I couldn't find a good way to check if arg is a Python3 string in Python2 because Python2 makes no distinction between byte literals and string literals.
Making the function only return true for basestring and str will yield different return values between Python2 and Python3 when passed in a bytes literal.